libs/forecasting/src/lib/ForecastingDashboard/Controls/ForecastComparison.tsx

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { Stack, Text } from "@fluentui/react";
+import {
+  ModelAssessmentContext,
+  defaultModelAssessmentContext,
+  BasicHighChart,
+  JointDataset,
+  orderByTime
+} from "@responsible-ai/core-ui";
+import { localization } from "@responsible-ai/localization";
+import { SeriesOptionsType } from "highcharts";
+import React from "react";
+
+import { forecastingDashboardStyles } from "../ForecastingDashboard.styles";
+
+import { getForecastPrediction } from "./getForecastPrediction";
+
+export class IForecastComparisonProps {}
+
+export interface IForecastComparisonState {
+  timeSeriesId?: number;
+  baselinePrediction?: number[];
+}
+
+const stackTokens = {
+  childrenGap: "l1"
+};
+
+export class ForecastComparison extends React.Component<
+  IForecastComparisonProps,
+  IForecastComparisonState
+> {
+  public static contextType = ModelAssessmentContext;
+  public context: React.ContextType<typeof ModelAssessmentContext> =
+    defaultModelAssessmentContext;
+
+  public constructor(props: IForecastComparisonProps) {
+    super(props);
+    this.state = {
+      baselinePrediction: undefined
+    };
+  }
+
+  public async componentDidMount(): Promise<void> {
+    const baselinePrediction = await this.getBaselineForecastPrediction();
+    if (baselinePrediction) {
+      this.setState({ baselinePrediction });
+    }
+  }
+
+  public async componentDidUpdate(): Promise<void> {
+    // Check if the time series was changed.
+    // In that case, we need to update our state accordingly.
+    const currentlySelectedTimeSeriesId =
+      this.context.baseErrorCohort.cohort.getCohortID();
+    if (currentlySelectedTimeSeriesId !== this.state.timeSeriesId) {
+      const baselinePrediction = await this.getBaselineForecastPrediction();
+      this.setState({
+        baselinePrediction,
+        timeSeriesId: currentlySelectedTimeSeriesId
+      });
+    }
```

Comment:
You will probably need to handle api call serilization in following pr.
When user change time series before api returns data, and first api call takes longer than second one, we may end up with having wrong data/chart.

Reply:
Do we have an example of that somewhere?

Reply:
Checked: `AbortController` code will have an example. Will add in next PR 🙂